java/src/main/resources/scripts/onnxFacedetect.py

```diff
@@ -0,0 +1,27 @@
+import os
+import numpy as np
+from PIL import Image
+import torchvision.transforms as transforms
+import onnxruntime
+from matplotlib.image import imread
+dl_path = os.path.abspath("../data/facedetector/facedetector.onnx")
```

Comment:
Also, this path doesn't seem right to me. Should be `dl_path = os.path.abspath("java/src/main/resources/data/facedetector/facedetector.onnx")
`

Reply:
It able to refer either 
**os.path.abspath(./src/main/resources/data/facedetector/facedetector.onnx)**  ( Fixed like this)
 or
os.path.abspath("../data/facedetector/facedetector.onnx")

But not working as below.
 os.path.abspath("java/src/main/resources/data/facedetector/facedetector.onnx").